book/src/concepts/system.md

```diff
@@ -134,6 +134,41 @@ This system will make all entities with both a `Transform` with a positive y coo
 
 Cool! Now that looks like something we'll actually do in our games!
 
+### Getting entities that have some components, but not others
+
+There is a special type of `Storage` in specs called `AntiStorage`.
+You can invoke it using the "!" symbol when borrowing a normal `Storage`.
+It is used like this:
+
+```rust,no_run,noplaypen
+# extern crate amethyst;
+# use amethyst::ecs::{System, ReadStorage, WriteStorage};
+# use amethyst::core::Transform;
+# struct FallingObject;
+# impl amethyst::ecs::Component for FallingObject {
+#   type Storage = amethyst::ecs::DenseVecStorage<FallingObject>;
+# }
+use amethyst::ecs::Join;
+
+struct NotFallingObjects;
+
+impl<'a> System<'a> for NotFallingObjects {
+    type SystemData = (
+        WriteStorage<'a, Transform>,
+        ReadStorage<'a, FallingObject>,
+    );
+
+    fn run(&mut self, (mut transforms, falling): Self::SystemData) {
+        for (mut transform, _) in (&mut transforms, !&falling).join() {
```

Comment:
should not iterate over them unconditionally given the conditional write below

Reply:
@torkleyy I don't believe there is a conditional write there.